cumulus_library/cli.py

```diff
@@ -340,19 +282,31 @@ def run_cli(args: dict):
 
     # all other actions require connecting to the database
     else:
+        # if we're targeting a builder explicitly, we always want to
+        # run in drop mode
+        if builder := args.get("builder"):
+            drop_table = builder
+        else:
+            drop_table = args.get("drop_table")
+        if args.get("db_type") == "duckdb":
+            schema = "main"
```

Comment:
This feels like a little bit of duckdb specific logic escaping from database.py, but it's not upsetting.

Reply:
oh thank you, i was like 'i need to mention something' and couldn't remember what it was.

If we want to get this cat back in the bag, we'll need to undo the overloading we're currently doing, where raw schema name and duckdb file path are going into the same location. So we'd basically add one new CLI arg and then split how that logic is handled in some places.

if we want to do this, i can create a ticket for it, but i don't want to do it on this PR given the density.

Reply:
Hmm, sounds like you are talking a bigger change than I was - I'm not 100% on what your vision there is.

I was talking more like the DuckDB database backend simply ignoring the input schema arg and using `main` or a method like `DatabaseBackend.decide_schema_name(args)` or something - just anything to put this tiny business logic under the umbrella of the DuckDB-specific code. Ideally in my mind, nothing outside of `databases.py` every has to know DuckDB exists.

Reply:
Per offline discussion, leaving this as is for now, but logging a ticket: #258